### PR TITLE
feat: op -> ioutil

### PR DESCRIPTION
**Pull request by wangzitian0**
Fix the error:
```
➜  gocmt git:(master) go install
# github.com/cuonglm/gocmt
./main.go:110:11: undefined: os.WriteFile
note: module requires Go 1.16
```

### CODE REVIEW

**Comment by cuonglm** [comment]
@wangzitian0 Thanks for the PR!

Note that `os.WriteFile` is only added in go1.16 https://pkg.go.dev/os#WriteFile, so this is expected behavior. Given that go1.17 is going to be released in next month https://tip.golang.org/doc/go1.17, I'm inclined to not making this support go1.15 and before.

File: main.go [hunk 1]
```diff
@@ -20,6 +20,7 @@ import (
 	"fmt"
 	"go/format"
 	"go/token"
+    "io/ioutil"
```
Comment (cuonglm) [inline main.go:23]: Please run `gofmt`.

File: main.go [hunk 2]
```diff
@@ -107,7 +108,7 @@ func processFile(filename, template string, inPlace bool) error {
 		newBuf = tralingWsRegex.ReplaceAll(newBuf, []byte(""))
 		newBuf = newlinesRegex.ReplaceAll(newBuf, []byte("\n\n"))
 		if inPlace {
-			return os.WriteFile(filename, newBuf, defaultMode)
+			return ioutil.WriteFile(filename, newBuf, defaultMode)
```
Comment (cuonglm) [inline main.go:111]: Please add a todo:

```
# TODO: use os.WriteFile and drop support for go1.15 and before once go1.17 release.
```

**Review by cuonglm — CHANGES_REQUESTED** [review]

File: main.go (hunk 2 above)
Comment (wangzitian0) [reply main.go:112]: Done.

**Review by wangzitian0 — COMMENTED** [review]
I run the `gofmt -s -w .`

File: testdata/empty_var_block.go [hunk 3]
```diff
@@ -1,5 +1,5 @@
 package p
 
 var (
-	// Error
+// Error
```
Comment (cuonglm) [inline testdata/empty_var_block.go:4]: Please revert this change

File: testdata/invalid_file.go [hunk 4]
```diff
@@ -1,3 +1,3 @@
 package p
 
-var i := 1
```
Comment (cuonglm) [inline testdata/invalid_file.go:3]: Please revert this change

**Review by cuonglm — CHANGES_REQUESTED** [review]

**Comment by wangzitian0** [comment]
> Given that go1.17 is going to be released in next month https://tip.golang.org/doc/go1.17, I'm inclined to not making this support go1.15 and before.

If it was inclined, I think we should revert the change of upgrade to 1.16?

File: testdata/invalid_file.go (hunk 4 above)
Comment (wangzitian0) [reply testdata/invalid_file.go:3]: It was auto fmt by `gofmt -s -w .`

Comment (cuonglm) [reply testdata/invalid_file.go:3]: @wangzitian0 you should not run `gofmt -s -w` on `testdata` directory, just revert the change.

**Comment by cuonglm** [comment]
> > Given that go1.17 is going to be released in next month https://tip.golang.org/doc/go1.17, I'm inclined to not making this support go1.15 and before.
> 
> If it was inclined, I think we should revert the change of upgrade to 1.16?

We can just change the code to use `ioutil.WriteFile`, since when you still can use go1.15 to build it. Even you revert the change of upgrade to 1.16, then you still have to use `ioutil.WriteFile`.

File: testdata/invalid_file.go (hunk 4 above)
Comment (wangzitian0) [reply testdata/invalid_file.go]: Done.

**Review by cuonglm — APPROVED** [review]

**Comment by cuonglm** [comment]
@wangzitian0 Thank you!
